MEMBERSHIP.md

```diff
@@ -0,0 +1,65 @@
+
+Chiunque è ben accetto in community e anche a livello contributivo per l'aggiunta di informazioni al libro.
+
+I nuovi contributors dovrebbero essere accolti nella community dagli ambassador esistenti, aiutati con il workflow delle PR e guidati alla documentazione e ai canali di comunicazione pertinenti o working group di riferimento.
+
+## Contributors
+
+I Contributors sono membri costantemente attivi della community. Questi sono tenuti a rimanere attivi nella community per mantenere il livello.
+
+Questi dimostrano di aderire ai principi di questo documento, di avere familiarità con l'organizzazione del progetto, i ruoli, le politiche, le procedure, le convenzioni, ecc. Le aspettative, le responsabilità e i requisiti specifici dei ruoli sono elencati di seguito.
+
+**Definito da**: Membro dell'organizzazione GitHub di Il Libro Open Source in triage e nel team `@Il-Libro-Open-Source/Contributor` e l'ingresso nella chat Telegram della community dedicata ai Contributor.
+
+### Requisiti
+
+- Abilitazione dell'autenticazione a due fattori sul proprio account GitHub
+- Hanno apportato molteplici contributi ai progetti o alla community. Il contributo può includere, ma non è limitato a:
+  - Autorizzare o revisionare PR su GitHub
+  - Aiutare o commentare le issues su GitHub
+  - Contribuire al progetto o alle discussioni della community (e.g. riunioni, GitHub discussions)
+- Aver letto la linee guida per le community, le regole e il documento sui Contributor
+- Essere sponsorizzato da 2 (da valutare il numero) Ambassador attivi
+- Aprire un issue su questo repository utilizzando il template
+  - Assicuratevi che i vostri sponsor siano @menzionati sull'issue aperto così che possano rispondere
+  - Gli sponsor dovranno rispondere con un :+1: alla issue per accettare l'ingresso dell'utente come ambassador
+- Tutti gli ambassador saranno comunque liberi di intervenire nella discussione commentando e dando una loro personale visione che sia essa positiva o negativa
+
+### Responsabilità e privilegi
+
+- Accesso all'organizzazione GitHub in triage (ovvero con permessi di sola lettura)
+- Rispondere ai problemi e alle relative PR o issue
+- Dare assistenza a nuovi membri della community
+- Contribuire attivamente alla scrittura di capitoli interni al libro Open Source
+- Gli Ambassador possono approvare le PR aperte
+- Accesso al canale Telegram dei contributors
+- Poter richiedere di diventare Ambassador
+
+# Ambassadors
+
+Gli Ambassadors sono membri costantemente attivi della community. Questi membri sono già dei contributor molto attivi e che desiderano prendere delle responsabilità ulteriori nei riguardi del progetto.
+
+**Definito da**: Inserimento nel Team Ambassador di GitHub con i poteri di write sull'organizzazione e accesso al team GH `@Il-Libro-Open-Source/Ambassadors`.
+
+### Requisiti
+
+- Contributor da almeno 3 mesi
+- Contributor attivo di almeno 1 capitolo del Libro
+- Potrebbe autonominarsi o venir nominato da altri ambassador
```

Comment:
```suggestion
- Potrebbe autonominarsi o venir nominato da altri ambassador
```

Qui in realtà attualmente abbiamo ancora il concetto di votazione pubblica che ancora non abbiamo sostituito. Al momento terrei quella e nel caso dopo aver portato dentro questa guideline proporrei un'alternativa, cosa ne pensi? Magari potremmo dire che la votazione è aperta esattamente come quando avevamo le porte aperte a nuovi Ambassador, ma cambiamo le linee guida dicendo che per proporsi come ambassador bisogna essere già contributor, cosa te ne pare?

Reply:
Sicuramente per diventare Ambassador devi essere già Contributor come definito dal requisito sotto.

La mia cosa era intesa, voglio passare ad essere Ambassador, ok ti puoi proporre come tale o uno già presente potrebbe consigliarti come possibile Ambassador perchè hai contribuito parecchio!

In realtà avendo i Contributor forse potrebbe rimanere sempre aperta la possibilità di scalare come Ambassador

Reply:
Aaaah ok, capito. A questo punto aggiungerei che una volta che la nomina è partita valgono le condizioni vigenti in quel momento per la nomina.

Però più che auto-nominarsi credo abbia più senso mantenere il concetto della sponsorship, al quale segue la votazione.

Cioè.. 2 Ambassador ti nominano per il passaggio da Contributor ad Ambassador e a quel punto parte la votazione, cosa ne dici?

Reply:
Su questo se ne può discutere, alla fine è un po' un misto alla fine la volontá deve essere condivisa con la persona e comunque si va sponsorizzato anche in questo caso! Quindi alla fine potresti autonomamente chiedere di sponsorizzarti da qualcuno

Reply:
Ok allora semplicemente proverei a riformularla, così sembra che uno possa auto nominarsi ambassador, invece può auto-candidarsi e a quel punto valgono le regole X e Y.